resources/public/api/conf/1.0/examples/create-contact-details-response.json

```diff
@@ -1,3 +1,33 @@
 {
-  "TODO": "TODO"
+  "contactDetails": [
+    {
+      "code": 9,
+      "detail": "07123 987654",
+      "type": "MOBILE TELEPHONE"
+    },
+    {
+      "code": 8,
```

Comment:
Code is unneeded here - none o fthe OGDs ask for this, same in create.

Reply:
For some reason I thought we filtered on this but its type.

Reply:
@parkeradam 

https://github.com/hmrc/individuals-details-api/blob/master/conf/application.conf#L245

https://github.com/hmrc/individuals-details-api/blob/master/conf/application.conf#L245

Looks required in Details?